### PR TITLE
Add bus client configuration

### PR DESCRIPTION
**Pull request by imakunin**

### CODE REVIEW

File: pkg/ytconfig/generator.go [hunk 1]
```diff
@@ -301,8 +301,7 @@ func (g *NodeGenerator) fillBusServer(c *CommonServer, s *ytv1.RPCTransportSpec)
 		c.BusServer = &BusServer{}
 	}
 
-	// FIXME(khlebnikov): some clients does not support TLS yet
-	if s.TLSRequired && s != g.commonSpec.NativeTransport {
```
Comment (koct9i) [inline pkg/ytconfig/generator.go:305]: We are not ready to remove this kluge.
Before making TLS fully required we need at least e2e test for that.

**Review by l0kix2 — COMMENTED** [review]
There are (canonized) tests for ytserver configs in this package. 
This change is either fail them and need to recanonize or if pr doesn't change existing configurations (all inputs lead to new fields be empty) we'll need new test case when new fields wouldn't be empty.

**Review by koct9i — CHANGES_REQUESTED** [review]

File: pkg/ytconfig/queue_agent.go [hunk 2]
```diff
@@ -13,6 +13,8 @@ type QueueAgent struct {
 
 type QueueAgentServer struct {
 	CommonServer
+
+	BusClient  *Bus       `yson:"bus_client,omitempty"`
```
[the remark this answers is not in the document]
Comment (imakunin) [reply pkg/ytconfig/queue_agent.go:17]: I got this list from @psushin

[the remark this answers is not in the document]
Comment (psushin) [reply pkg/ytconfig/queue_agent.go:17]: Because bus_client is configured separately only for those components, that do not rely on cluster_connection (or some parts of them do not rely) - such as disovery, master_connector or master itself.

[the remark this answers is not in the document]
Comment (imakunin) [reply pkg/ytconfig/queue_agent.go:17]: After Pasha's answer, it looks like it's not worth generalizing.

[the remark this answers is not in the document]
Comment (achulkov2) [reply pkg/ytconfig/queue_agent.go:17]: yeah, ok

**Review by koct9i — APPROVED** [review]

**Review by l0kix2 — APPROVED** [review]

**Comment by l0kix2** [comment]
@psushin can you check last version of changes and if it looks ok — let's merge

**Comment by psushin** [comment]
Ship it!

**Review by achulkov2 — APPROVED** [review]
